### PR TITLE
Change reference to nunit.framework.dll

### PR DESCRIPTION
**Pull request by van800**
Original reference is 
C:/Program Files/Unity 2017.2.0b3/Editor/Data/UnityExtensions/Unity/TestRunner/net35/unity-custom/nunit.framework.dll
Updated one is 
C:/Program Files/Unity 2017.2.0b3/Editor\Data/Managed/nunit.framework.dll
See comparison 
https://youtrack.jetbrains.com/_persistent/image2.png?file=74-435202&c=true&updated=1501665019076

Original one has System.Web.UI type, and thus doesn't play fine with Re# unit test runner.

### CODE REVIEW

**Comment by van800** [comment]
Fix helps only on Windows, because on mac there is other problem, for which I haven't found a workaround. RSRP-459932

**Comment by van800** [comment]
Compilation against different Unity versions is ensured https://buildserver.labs.intellij.net/viewLog.html?buildId=20663281&tab=buildResultsDiv&buildTypeId=ReSharperUnity_ValidateUnity3dRiderBuilds

File: resharper/src/resharper-unity/Unity3dRider/Assets/Plugins/Editor/JetBrains/RiderAssetPostprocessor.cs [hunk 1]
```diff
@@ -68,14 +67,52 @@ private static void UpgradeProjectFile(string projectFile)
       XNamespace xmlns = projectContentElement.Name.NamespaceName; // do not use var
 
       FixTargetFrameworkVersion(projectContentElement, xmlns);
+      FixSystemXml(projectContentElement, xmlns);
```
Comment (IvanPashchenko) [inline resharper/src/resharper-unity/Unity3dRider/Assets/Plugins/Editor/JetBrains/RiderAssetPostprocessor.cs]: this helps on Linux

Comment (IvanPashchenko) [inline resharper/src/resharper-unity/Unity3dRider/Assets/Plugins/Editor/JetBrains/RiderAssetPostprocessor.cs]: image

**Pull request merged**